src/overlays/actors/ovl_En_Xc/z_en_xc.c

```diff
@@ -280,19 +282,26 @@ void func_80B3C9EC(EnXc* this) {
 void func_80B3CA38(EnXc* this, PlayState* play) {
     // If Player is adult but hasn't learned Minuet of Forest
     if (!GET_EVENTCHKINF(EVENTCHKINF_50) && LINK_IS_ADULT) {
+        s32 pad;
+
         this->action = SHEIK_ACTION_INIT;
     } else {
         Actor_Kill(&this->actor);
     }
 }
 
 s32 EnXc_MinuetCS(EnXc* this, PlayState* play) {
+    Player* player;
+    f32 z;
+
```

Comment:
I would be a little bit more descriptive like `playerPosZ`. I understand its used right after its assignment but still think its best